buildkite/bazelci.py

```diff
@@ -1375,7 +1448,7 @@ def test_logs_for_status(bep_file, status):
     return targets
 
 
-def execute_command_and_get_output(args, shell=False, fail_if_nonzero=True):
+def execute_command_and_get_output(args, shell=False, fail_if_nonzero=True, print_output=True):
```

Comment:
We could add a parameter `capture_stderr=True` here and then change below line to:

```
...
stderr=subprocess.PIPE if capture_stderr else None
...
```

Then we wouldn't have to filter output for targets in `expand_test_target_patterns` and could replace that with a simple `output.split("\n")`.

Reply:
Done